src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -298,6 +298,24 @@ private bool DerivesFrom(string typea, string typeb, ProtoCore.Core core)
             }
         }
 
+        /// <summary>
+        /// For checking the ML Context menu item
+        /// </summary>
+        public bool IsML
+        {
+            get { return dynamoViewModel.PreferenceSettings.DefaultNodeAutocompleteSuggestion == Models.NodeAutocompleteSuggestion.MLRecommendation; }
+            set { }
+        }
+
+        /// <summary>
+        /// For checking the Object Type Context menu item
+        /// </summary>
+        public bool IsObjectType
```

Comment:
I would remove this property for now, since we only have two choices for now

Reply:
Done.